tsconfig.json

```diff
@@ -7,10 +7,8 @@
     "moduleResolution": "node",
     "types": ["node", "mocha"],
     "typeRoots": ["node_modules/@types"],
-    "module": "commonjs"
+    "module": "commonjs",
+    "declaration": true
```

Comment:
@thebigredgeek don't you also have to add `"types": "index.d.ts"` to `package.json`?

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package